staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -443,7 +443,7 @@ func TestPreShutdownHooks(t *testing.T) {
 		var r result
 		client := newClient(true)
 		for i := 0; i < 5; i++ {
-			r = doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), 100*time.Millisecond)
+			r = doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), 1*time.Second)
```

Comment:
looks like a random number. Why not 2.7s or 370ms?

Reply:
it is a max timeout and is the same value is used on the other tests ... carrying over with current state ... if something keeps flaking here I will do a more thoroughly investigation, but right now it is blocking the CI

Reply:
```
rep -r doer.Do ./staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go | grep time
                resultGot := doer.Do(connReusingClient, shouldReuseConnection(t), "/echo?message=request-on-an-existing-connection-should-succeed", time.Second)
                resultGot = doer.Do(newClient(true), shouldUseNewConnection(t), "/echo?message=request-on-a-new-tcp-connection-should-succeed", time.Second)
                resultGot := doer.Do(connReusingClient, shouldReuseConnection(t), "/echo?message=request-on-an-existing-connection-should-succeed", time.Second)
                resultGot = doer.Do(newClient(true), shouldUseNewConnection(t), "/echo?message=request-on-a-new-tcp-connection-should-succeed", time.Second)
        resultGot := doer.Do(connReusingClient, func(httptrace.GotConnInfo) {}, "/readyz", time.Second)
                resultGot := doer.Do(connReusingClient, func(httptrace.GotConnInfo) {}, "/readyz", time.Second)
        resultGot = doer.Do(newClient(true), shouldUseNewConnection(t), "/echo?message=request-on-a-new-tcp-connection-should-fail-with-429", time.Second)
        resultGot = doer.Do(connReusingClient, shouldReuseConnection(t), "/echo?message=request-on-an-existing-connection-should-fail-with-429", time.Second)
                        r = doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), time.Second)
                result := doer.Do(client, func(httptrace.GotConnInfo) {}, fmt.Sprintf("/echo?message=attempt-%d", i), 100*time.Millisecond)
```

having a subsecond timeout  for an http request is really a very low value

Reply:
agreed… CI environments are often either underpowered or overloaded… subsecond timeouts are usually too tight tolerances

Reply:
I'd recommend lengthening this to something like the normal 10 seconds if we're not making a request that we expect to hit the timeout. I don't see a benefit to making this a short timeout and maybe flaking on it.